packages/aws-cdk/lib/api/cxapp/stacks.ts

```diff
@@ -225,6 +226,22 @@ export class AppStacks {
     }
   }
 
+  public getTagsFromStackMetadata(stack: SelectedStack): Tags {
+    const tags = [];
+    for (const id of Object.keys(stack.metadata)) {
+      const metadata = stack.metadata[id];
+      for (const entry of metadata) {
+        switch (entry.type) {
+          case "Tags":
```

Comment:
Define the key in @aws-cdk/cx-api and use a "aws:cdk" prefix.

Reply:
Does the "aspect/no aspect" question have any implication on this? Happy to change this (after the many meetings I got today)